src/imap_db/migrations/versions/2025_07_30-abd9fd48d848_added_creation_modified_and_deleted_.py

```diff
@@ -0,0 +1,36 @@
+"""Added creation, modified and deleted dates
+
+Revision ID: abd9fd48d848
+Revises: 17c951b7b5c3
+Create Date: 2025-07-30 09:12:30.147100
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "abd9fd48d848"
+down_revision = "17c951b7b5c3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column("files", "date", nullable=False, new_column_name="content_date")
+    op.add_column("files", sa.Column("creation_date", sa.DateTime(), nullable=False))
+    op.add_column(
+        "files", sa.Column("last_modified_date", sa.DateTime(), nullable=False)
+    )
```

Comment:
Good practise if the database also enforces the timestamps automatically, see https://stackoverflow.com/a/33532154/3140853 for ways to do this

Maybe this would work?
```
from sqlalchemy.sql import func

    op.add_column("files", sa.Column("creation_date", sa.DateTime(), nullable=False, server_default=func.now()))
    op.add_column(
        "files", sa.Column("last_modified_date", sa.DateTime(), nullable=False, onupdate=func.now())
    )
```